datumaro/plugins/open_images_format.py

```diff
@@ -269,7 +269,7 @@ def load_from(annotation_name):
                     if _RE_INVALID_PATH_COMPONENT.fullmatch(subset):
                         raise UnsupportedSubsetNameError(item_id=image_id, subset=subset)
 
-                    image_path = image_paths_by_id.get(image_id)
+                    image_path = image_paths_by_id.get(image_id, image_id)
 
                     image = None
                     if image_path is not None:
```

Comment:
Now, this condition is never False, and then `image.path` is set to an incorrect value.

Reply:
Changed this solution